bibliopixel/drivers/__init__.py

```diff
@@ -0,0 +1 @@
+from . channel_order import ChannelOrder
```

Comment:
Don't disagree... but how else do we make ChannelOrder available without having to fully qualify it? Also, we have to do these to make some import paths compatible with older BP2 code. I'd be happy to get rid of all this, but right now that's a something that's possibly going to break a lot of stuff and be a pain to test. I've logged issue #231 and assigned to 3.x milestone. Yes, we should do it, but no important enough now.